integration/manifests/svcwatcher/0svcwatcher_rbac.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: svcwatcher
+  namespace: kube-system
+  labels:
+      kubernetes.io/cluster-service: "true"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
+  name: system:svcwatcher
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  verbs:
+  - list
+  - watch
+  - get
+  - update
+  - create
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - services
```

Comment:
svcwatcher also needs DanmNet,DanmEp, and EndPoints as far as I know

Reply:
I think we have those also in this ClusterRole. Danm stuff starting in line 37 with full control and also the endpoints  above in line 19

Reply:
ah my bad, I still didn't have my morning coffee :)